packages/office-ui-fabric-react/src/components/Check/FastCheck.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+import { IFastCheckProps, ICheckStyleProps, ICheckStyles } from './Check.types';
+import { getStyles } from './Check.styles';
+import { FontIcon } from '../../Icon';
+import { classNamesFunction, styled } from '../../Utilities';
+
+const getClassNames = classNamesFunction<ICheckStyleProps, ICheckStyles>();
+
+/**
+ * Faster check component. Still in development, so props are subject to change.
+ * @beta
+ */
+export const FastCheckBase: React.StatelessComponent<IFastCheckProps> = props => {
+  const { checked = false, className, theme, styles } = props;
+
+  const classNames = getClassNames(styles!, { theme: theme!, className, checked });
+
+  return (
+    <div className={classNames.root}>
+      <FontIcon iconName="CircleRing" className={classNames.circle} />
+      <FontIcon iconName="StatusCircleCheckmark" className={classNames.check} />
+    </div>
+  );
+};
```

Comment:
This seems pretty similar to `Check.base`.Is there no way to avoid duplicating `Check` impl and select the type of icon component to use?  If props are different this could just be a re-export of Check with new typing. (Unless there are other differences I'm not seeing...)

Reply:
The existing Check.base is a class (for no good reason), so technically making it a function is a breaking change (but will only break people if they're extending it, which would be a bit weird in this case). If other people think it's okay I can add a prop for selecting the icon component.